bower.json

```diff
@@ -13,6 +13,7 @@
     "normalize-scss": "^3.0.3",
     "flickity": "^2.0.5",
     "babel-polyfill": "^0.0.1",
+    "pdfjs-embed": "eScholarship/pdfjs-embed",
```

Comment:
Yes! same way as package.json, by adding #<branch>. Change this and will be adding it as next commit momentarily.